docs/tokens.md

```diff
@@ -28,6 +28,18 @@ as Python, Ruby and Bash. These are simply discarded by a Monogram processor.
     - For example `0x1.1e2` = 272.0. 
     - Note that both the radix part and the exponent part are written in
```

Comment:
Why is this? I've tried to find an explanation searching online, but couldn't find anything. Is it just accepted practice?

Reply:
That's a very good question. There is literally only one programming language that generalised numeric literals this far - and that is Pop-11. This choice follows the lead of Pop-11 in writing the exponent in decimal notation, although the base remains the same as the radix (also written in base 10). It is a niche case, and there are arguments both ways. I did consider allow exponents to have their own base (!) but although cute it is giving too much machinery to a case that might never be used in practice.